influxql/result.go

```diff
@@ -35,6 +35,22 @@ func (t *TagSet) Swap(i, j int) {
 	t.Filters[i], t.Filters[j] = t.Filters[j], t.Filters[i]
 }
 
+func (t *TagSet) Reverse() *TagSet {
```

Comment:
Does this need to return a new `TagSet`?  Seems like it could just reverse the existing one to avoid the extra allocations.